packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -117,25 +117,35 @@ export const ConsentBannerText = styled.p`
 // Style `button` and `a` as children due to inability to set `on`
 // prop on styled component as required for the amp useage
 const ListItem = styled.li`
+  text-align: center;
+  display: inline-block;
+  width: 17.3125rem;
```

Comment:
again maybe extract a constant linked named after the gel guideline this dervied from maybe this aligns with 'spacing units' https://www.bbc.co.uk/gel/guidelines/spacing-units ?

Reply:
This is for the width of the button which in the design is 277px, I checked the GEL spacing units and breakpoints and I couldn't find it, i'm not sure if it's derived from a breakpoint or from spacing units.